zebra/zebra_nhg.c

```diff
@@ -3435,16 +3435,22 @@ bool zebra_nhg_proto_nexthops_only(void)
 }
 
 /* Add NHE from upper level proto */
-struct nhg_hash_entry *zebra_nhg_proto_add(uint32_t id, int type,
```

Comment:
let's get this commit into it's own PR and just get it in.  It's not in any way going to ever be controversial.